### DISCUSSION TITLE
Where did the subgraph path go for the subgraph node?

### DISCUSSION THREAD

**Discussion by LucidTal** [nxt-dev/nxt_editor#175] [Q&A] [answered]
Hi all!

I just finished the tutorial for nxt and realized that the subgraph path was missing for my subgraph node.

Tutorial view:
![image](https://user-images.githubusercontent.com/79533416/108880049-e4120b80-75cf-11eb-8c17-6e29f2aebe8b.png)

Subgraph node at my workstation:
![image](https://user-images.githubusercontent.com/79533416/108880182-07d55180-75d0-11eb-835b-63e0d3a32b40.png)

Did it possibly get moved somewhere else?

Thank you!

**Comment by ImLucasBrown** [comment, marked as answer]
Hey @LucidTal,
Sorry for the confusion, that tutorial was recorded using a still unreleased update. 
You can find the sub-graph path attribute in the attribte table under the name `_graph_path`.

![image](https://user-images.githubusercontent.com/54835354/108887332-1d4e7980-75d8-11eb-9608-4720c02ded57.png)

As of right now, we don't have a firm release date for the update shown in the video. However the functionality shown does still work, even though the UI looks a little different.

**Reply by LucidTal** [reply]
Thanks so much Lucas! The way the tutorial's build was able to bring up all the of the stage's attributes to make local looks like it is going to be really really useful! I realized after posting I could manually make the attributes on the instanced subgraph node to talk to the referenced graph's stage attributes. We're testing this out at work right now and it's really intuitive and powerful. 

Much obliged!

**Comment by ordinc** [comment]
@LucidTal @ImLucasBrown 
How to realize the operation of "Preview sub-graph STAGE attrs"?
![image](https://user-images.githubusercontent.com/35759494/131079933-76408956-a11c-4179-b9e7-051d85f4357a.png)

**Reply by ImLucasBrown** [reply]
This is a feature @MichaelAldrich has been working on after his day job, I don't think we have a time estimate on when it will be officially released. We wish all these cool features were quick to release, but NXT is our side project we only get to work on it after our day jobs.
